### ISSUE TITLE
Unable to find a way to change default icon on AnimatedIcon

### ISSUE THREAD

**Issue by Nik99** [flutter/flutter#21355]
Is it a bug, a missing doc / parameter required or simply isn't implemented a way to change the default icon ? 

If is not implemented, will this be implemented ? It would be very useful or at least, if not all icons a list of pre-done icons for the moment, like settings, menu (already default) ecc.

**Comment by zoechi** [comment]
Can you please elaborate an what you try to accomplish or why you need this?
Please also add the output of `flutter doctor -v`.

**Comment by Nik99** [comment]
I'm trying to make a backdrop for settings like the one listed in the gallery, but the opening button defaults to the menu icon, i can only specify the closing icon. What i want is the ability to specify all material icons as a starter icon (default viewed) or at least, have a choice of 20-30 icons to choose if not all icons are available to the animation.

**Comment by Nik99** [comment]
I'm trying to make a backdrop for settings like the one listed in the gallery, but the opening button defaults to the menu icon, because with the class AnimatedIcon i can only specify the closing icon. What i want is the ability to specify all material icons as a starter icon (default viewed) or at least, have a choice of 20-30 icons to choose if not all icons are available to the animation.

flutter doctor -v
[✓] Flutter (Channel beta, v0.6.0, on Mac OS X 10.13.6 17G65, locale it-IT)
    • Flutter version 0.6.0 at /Users/nicolas/flutter
    • Framework revision 9299c02cf7 (3 weeks ago), 2018-08-16 00:35:12 +0200
    • Engine revision e3687f70c7
    • Dart version 2.1.0-dev.0.0.flutter-be6309690f

[✓] Android toolchain - develop for Android devices (Android SDK 28.0.2)
    • Android SDK at /Users/nicolas/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-28, build-tools 28.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1024-b01)
    • All Android licenses accepted.

[✓] iOS toolchain - develop for iOS devices (Xcode 9.4.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 9.4.1, Build version 9F2000
    • ios-deploy 1.9.2
    • CocoaPods version 1.5.3

[✓] Android Studio (version 3.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 26.0.1
    • Dart plugin version 173.4700
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1024-b01)

[✓] VS Code (version 1.26.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 2.18.0

• No issues found!

**Comment by zoechi** [comment]
Did you accidentally post twice? Seems to be the same content. You can delete one with the [...] buton on the top-right of the message.

**Comment by Nik99** [comment]
@zoechi Yes, sorry, i hided the devices IDS and ended up to delete the first message in order to hide them. Thanks for the support! Will this be implemented soon or is it not in priorities ?
